docs/modules/sql/pages/user-defined-types.adoc

```diff
@@ -591,11 +481,9 @@ Replacing nested column value:
 UPDATE users SET organization = ((organization).id, (organization).name, ((organization).office.id, 'new-office-name'))
 ----
 
-NOTE: Updating UDT-based columns requires providing a value for every column in the UDT and its child UDTs, however
-`null` can also be specified in place of nested UDT column to initialize it to `null`. Not providing full list of columns
-will cause a query validation error.
+NOTE: Updating UDT columns requires providing a value for every column in the UDT and its child UDTs unless it needs to be set to `null`. Not providing full list of columns will cause a query validation error.
```

Comment:
```suggestion
NOTE: To update UDT columns, a value must be provided for every column in the UDT and its child UDTs unless it needs to be set to `null`. If a full list of columns is not provided, a query validation error occurs.
```

Reply:
"A full list" sounds very strange.

Reply:
Does a "partial list" sound strange?

Also, I didn't change that, that was from your original text

Reply:
Oh, hang on, reading that again, it is the use of the indefinite article that you are questioning. 

Nah, you need it, whether it sounds strange to you or not.

Reply:
90767538